docs/structured.md

```diff
@@ -166,6 +166,13 @@ A row insertion involves computing the primary and secondary keys for the row,
 writing the data under the primary key, and writing the primary key to all the
 secondary keys. A row deletion behaves analogously.
 
+**Row sentinel and NULL column values**
```

Comment:
can this go up into the Primary key addressing section? it seems like a caveat of that section that comes surprisingly late